src/Orleans.Core/Configuration/GlobalConfiguration.cs

```diff
@@ -840,9 +827,9 @@ internal override void Load(XmlElement root)
                             ServiceId = ConfigUtilities.ParseGuid(child.GetAttribute("ServiceId"),
                                 "Invalid Guid value for the ServiceId attribute on the Azure element");
                         }
-                        if (child.HasAttribute("DeploymentId"))
+                        if (child.HasAttribute("ClusterId"))
                         {
-                            DeploymentId = child.GetAttribute("DeploymentId");
+                            this.ClusterId = child.GetAttribute("ClusterId");
```

Comment:
This is a breaking change, xml config now need to change its "DeploymentId" to "ClusterId", is this intentional? @jdom

`ClientConfiguration` parse DeploymentId as ClusterId, but silo doesn't. This is inconsistent. Is this intentional?

Reply:
oops, no... I explicitly told resharper not to change this one :/ I guess it insisted until he won. Nice catch. My intention is that XML parsing stays the same for legacy

Reply:
Thanks, addressed in #3736